tests/lib/parse.js

```diff
@@ -78,6 +78,93 @@ describe("parse()", () => {
             assert.deepStrictEqual(tester.getRaw(ast), require("../fixtures/parse/all-pieces.json"));
         });
 
+        it("should output the same value for program.start, end and range when there is a leading/trailing comments", () => {
+            const ast = espree.parse("/* foo */ bar /* baz */", {
+                range: true
+            });
```

Comment:
Can we add another test like this for `Program`, but without `range: true` and `loc: true`?